### PR TITLE
heatmap doesn't show up in md but is plotted

### PR DESCRIPTION
**Pull request by sgosline**
I'm interested to hear your thoughts on the three figures generated. Not much commentary in the MD.

### CODE REVIEW

File: figures/Fig3ideas.Rmd [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+rf.results<-synTableQuery('select * from syn21326918')$asDataFrame()%>%
+  select(-c(ROW_ID,ROW_VERSION))
+
+DT::datatable(rf.results)
+
+```
+
+## Figure 3A
+
+Now we can go ahead and plot a venn diagram of the RF results. The first thing we want to do is evaluate how many latent variables predict which tumor types. This Venn Diagram illustrates that there are numerous LVs that predict many different tumor types.
```
Comment (allaway) [inline figures/Fig3ideas.Rmd]: this paper is a great Vennue for this sort of plot

Comment (sgosline) [reply figures/Fig3ideas.Rmd]: lolol

**Comment by sgosline** [comment]
Added the gap column.

**Comment by allaway** [comment]
https://media.giphy.com/media/QyrysGo7Hz70I/giphy.gif LGTM!

**Pull request merged**